torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -1342,6 +1353,7 @@ def perspective_bounding_box(
     perspective_coeffs = _perspective_coefficients(startpoints, endpoints, coefficients)
 
     original_shape = bounding_box.shape
+    # TODO: first cast to float if bbox is int64 before convert_format_bounding_box
```

Comment:
Is this still valid?

Reply:
Yes, this is new and is still valid. Here the conversion can reduce bbox precision in case of int64 CXCYWH.

Reply:
So, do we need to fix this before merge or is it fine to leave for later?

Reply:
Yes, later with a follow up PR